TransIT.API/Controllers/EmployeeController.cs

```diff
@@ -106,7 +112,76 @@ public async Task<IActionResult> Create([FromBody] EmployeeDTO employeeDTO)
             }
             catch (Exception e)
             {
-                throw e;
+                _logger.LogError(e, string.Empty);
+                return StatusCode(500, new ExtendedErrorDTO(e));
+            }
+        }
+
+        [HttpGet("boardnumber/{number}")]
+        public async Task<IActionResult> GetByBoardNumber(int number)
+        {
```

Comment:
I don't see changes: add try catch for **ALL** endpoints you've added, return 500 in case of exception, log the exception using logger